task-sdk/tests/task_sdk/execution_time/test_supervisor.py

```diff
@@ -1524,6 +1526,11 @@ def test_handle_requests(
         next(generator)
         msg = message.model_dump_json().encode() + b"\n"
         generator.send(msg)
+
+        if isinstance(message, GetVariable):
+            mock_mask_secret.assert_called_once()
+            mock_mask_secret.assert_called_with("test_value", "test_key")
```

Comment:
```suggestion
            mock_mask_secret.assert_called_with("test_value", "test_key")
```

We don't need both asserts.

Reply:
Yeah let me simplify this further.